docs/src/pages/getting-started/installation/troubleshooting.react.mdx

```diff
@@ -57,10 +57,6 @@ When working with a [Vite](https://vitejs.dev) project you must make a few modif
 ...
   <script>
     window.global = window;
-    window.process = {
```

Comment:
I may need to do this later after release, otherwise it will be updated as soon as I merge to master. But I wanted to show that it can be removed after this fix!